tendrl/monitoring_integration/__init__.py

```diff
@@ -0,0 +1,75 @@
+import __builtin__
+import maps
+import traceback
+import sys
+import json
+from grafana.connection import get_conf
+from grafana.dashboard import create_dashboard
+from grafana.dashboard import set_home_dashboard
+from grafana.dashboard import get_dashboard
+from grafana.dashboard import get_all_dashboards
+from grafana.datasource import create_datasource
+from grafana.exceptions import ConnectionFailedException
+
+
+def main():
+
+    dashboards = []
+    config = get_conf("../../grafana.conf.yml")
```

Comment:
Based on my comment below, you will now need to load the "/etc/tendrl/monitoring-integration/monitoring-integration.conf.yml"   instead of loading ../../grafana.conf.yml and graphite.conf.yml